bigsea-scaler/service/api/controller/plugins/proportional/proportional_alarm.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, actuator, metric_source, trigger_down, trigger_up, min_cap, m
         self.min_cap = min_cap
         self.max_cap = max_cap
         self.metric_rounding = metric_rounding
-        self.heuristic_options = heuristic_options
+        self.heuristic_options_1 = heuristic_options
```

Comment:
This variable name should be something more detailed, this _1 doesn't make much sense